examples/gcp/00-ark-config.yaml

```diff
@@ -20,11 +20,13 @@ metadata:
   name: default
 persistentVolumeProvider:
   gcp:
+    credentialsFile: /credentials/cloud
```

Comment:
Yes and no. The PVProvider doesn't use it directly; it needs to have `GOOGLE_APPLICATION_CREDENTIALS` set to this path. I feel like this is getting a bit messy... In my branch now, you could have your backup provider be something other than GCE, but if you want to handle PVs, then you have to specify the credentialsFile for the gcp PVProvider, because the code looks for that and sets the env var for you (so you don't have to put it in the deployment.yaml).

I feel like this is getting messy and I'm getting more and more in favor of having a config that looks like this, although I know @jbeda doesn't like it:

```yaml
clouds:
  myCoolGCP:
    credentialsFile: /credentials/cloud
    project: foo
    zone: us-east1-b
  someOtherCloud:
    someDetails: goHere
persistentVolumeProvider:
  cloud: myCoolGCP
backupStorageProvider:
  cloud: someOtherCloud
  bucket: mybucket
```

This way, we load each cloud provider one time, instead of twice like we're doing now...

Let's discuss when you're in.